playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup/iam.tf

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+resource "google_service_account" "cloudbuild_service_account_id" {
+  account_id   = var.cloudbuild_service_account_id
+  display_name = var.cloudbuild_service_account_id
+  description  = "The service account cloud build will use to deploy Playground"
+}
+
+// Provision IAM roles to the IaC service account required to build and provision resources
+resource "google_project_iam_member" "cloud_build_roles" {
+  for_each = toset([
+    "roles/appengine.appAdmin",
```

Comment:
is this a Mimimal priviledge role set?

Reply:
Yes, same as in Sergey's PR, because Cloud Build SA will be deploying the same Playground, therefore same logic